.github/workflows/trivy.yml

```diff
@@ -17,7 +17,7 @@ env:
   MEMBER_AGENT_IMAGE_NAME: member-agent
   REFRESH_TOKEN_IMAGE_NAME: refresh-token
 
-  GO_VERSION: '1.22.3'
+  GO_VERSION: '1.22.4'

```

Comment:
why does this matter?

Reply:
it should not, keep it consistent with the go version when building the images.

Reply:
but our go mod is 1.22.3

Reply:
yeah, it should not matter though. I'm fine to keep it consistent with our go mod file. Either should be fine.